### PR TITLE
Con 1081 reviews are visible on product detail page

### PR DESCRIPTION
**Pull request by CarinaHoffmann**

### CODE REVIEW

File: product/action-creators/errorProductReviews.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ERROR_PRODUCT_REVIEWS } from '../constants';
```
Comment (richardgorman) [inline product/action-creators/errorProductReviews.js]: Please add a space between this line and the next.

File: product/action-creators/errorProductReviews.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Dispatches the ERROR_PRODUCT_REVIEWS action
```
Comment (richardgorman) [inline product/action-creators/errorProductReviews.js]: Punctuation: period at the end of the sentence.

File: product/action-creators/receiveProductReviews.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { RECEIVE_PRODUCT_REVIEWS } from '../constants';
+
+/**
+ * Dispatches the RECEIVE_PRODUCT_REVIEWS action
```
Comment (richardgorman) [inline product/action-creators/receiveProductReviews.js]: Punctuation: period.

File: product/action-creators/requestProductReviews.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { REQUEST_PRODUCT_REVIEWS } from '../constants';
+
+/**
+ * Dispatches the REQUEST_PRODUCT_REVIEWS action
```
Comment (richardgorman) [inline product/action-creators/requestProductReviews.js]: Punctuation: period.

File: product/action-creators/requestProductReviews.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+ * @param {string} productId The ID of the product
+ * @param {number} limit The max number of reviews
```
Comment (richardgorman) [inline product/action-creators/requestProductReviews.js]: Typo: max -> maximum

File: product/actions/getProductReviews.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PipelineRequest from '@shopgate/pwa-core/classes/PipelineRequest';
+import { logger } from '@shopgate/pwa-core/helpers';
+
```
Comment (richardgorman) [inline product/actions/getProductReviews.js]: Please remove spaces between imports.

File: product/actions/getProductReviews.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+import requestProductReviews from '../action-creators/requestProductReviews';
+import receiveProductReviews from '../action-creators/receiveProductReviews';
+import errorProductReviews from '../action-creators/errorProductReviews';
+
+/**
+ * Request product reviews for a product from server.
+ * @param {string} productId The product ID
+ * @param {number} limit The max number of reviews to fetch
```
Comment (richardgorman) [inline product/actions/getProductReviews.js]: Typo: max -> maximum

File: product/action-creators/errorProductReviews.js (hunk 1 above)
Comment (CarinaHoffmann) [reply product/action-creators/errorProductReviews.js]: done

File: product/action-creators/requestProductReviews.js (hunk 4 above)
Comment (CarinaHoffmann) [reply product/action-creators/requestProductReviews.js]: done

File: product/action-creators/errorProductReviews.js (hunk 2 above)
Comment (CarinaHoffmann) [reply product/action-creators/errorProductReviews.js]: Done.

File: product/action-creators/receiveProductReviews.js (hunk 3 above)
Comment (CarinaHoffmann) [reply product/action-creators/receiveProductReviews.js]: Done.

File: product/action-creators/requestProductReviews.js (hunk 5 above)
Comment (CarinaHoffmann) [reply product/action-creators/requestProductReviews.js]: Done.

File: product/actions/getProductReviews.js (hunk 6 above)
Comment (CarinaHoffmann) [reply product/actions/getProductReviews.js]: Done.

File: product/actions/getProductReviews.js (hunk 7 above)
Comment (CarinaHoffmann) [reply product/actions/getProductReviews.js]: Done.

**Pull request merged**